homework-2/webpack.config.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'main.js'
+  },
+  watchOptions: {
+    poll: true,
+  },
+  mode: 'development',
+  module: {
+    rules: [
+      {
+        test: /\.jsx?$/,
+        use: 'babel-loader',
+        exclude: '/node_modules/'
+      },
```

Comment:
```suggestion
      resolve: { extensions: [".js", ".jsx"] } },
```
Данное свойство добавьте в правило массива `rules` и вам не придется при подключении модуля указывать его расширение. 
Т.е. вместо `import App from "./App.jsx"` будете просто указывать `import App from "./App"`